<!DOCTYPE html>
<html>
<body>
<h1> Data and file types used for the plots </h1>
<a href="https://nickpowe.github.io/pharmacogene_rna_regulation_plots/info/example_plot.pdf" target="_blank"> example plot </a> <br> 
<h3> RNA-seq </h3>
  <p> Poly-A selected mRNA from single donor hepatocytes sequenced and processed to bdg (bedgraph) files for each gene. As an example, the CYP3A4 file is 8.2 MB. <p>
<h3> mir-eCLIP </h3>
    <p> chimeric mRNA-miRNA hybrids are sequenced and processed to bdg files for each gene for each microRNA. As an example, for CYP3A4 for Run 2, there is a total chimeric file that is 21 KB, and 187 other files each corresponding to a unique miRNA (largest is for miR-122-5p at 9.5 KB). these are present for each run of mir-eCLIP. </p>
<h3> mir-eCLIP runs </h3>
  <p> Run 1 is 100-donor pooled hepatocytes, experiment executed by Eclipse Bioinnovations. </p>
  <p> Run 2 is individual-donor hepatocytes, experiment executed by Eclipse Bioinnovations. </p>
  <p> Run 3 is individual-donor hepatocytes, experiment executed by Skaar lab using Eclipse Kit. </p>
  <p> Run 4 is individual-donor hepatocytes, experiment executed by Eclipse Bioinnovations. </p>
  <p> Run 5 is individual-donor hepatocytes, experiment executed by Eclipse Bioinnovations, using gene-specific pulldowns for CYP3A4, PXR (NR1I2), CYP2B6, CYP2C19, and CYP2D6 (Immunoprecipitation library 1). </p>
  <p> Run 6 is individual-donor hepatocytes, experiment executed by Eclipse Bioinnovations, using gene-specific pulldowns for CYP3A4, PXR (NR1I2), CYP2B6, CYP2C19, and CYP2D6 (Immunoprecipitation library 2). </p>
  <p> Run 7 is individual-donor hepatocytes, experiment executed by Skaar lab using Eclipse Kit. </p>
  <p> Run 8 is individual-donor hepatocytes, experiment executed by Skaar lab using Eclipse Kit. </p>
<h3> Seed Matching </h3>
  <p> The seed region of each microRNA is tested for complementarity with every 6 adjacent nucleotides on a sliding scale of 1 interval. the number of complementarity (0/6, 1/6, 2/6, 3/6, 4/6, 5/6, 6/6) is recorded at the 4th position (meaning it encompases the preceeding 3 nucleotides, and the following 2). 6/6 seedmatch means all nucleotides were complementary. These data are stored in a txt file for each gene, with columns for each microRNA. the CYP3A4 file is 3.3 MB   </p>
<h3> SHAPE </h3>
  <p> the shape data detects the interactiveness of each nucleotide position within mRNA. a value extending further below the x-axis indicates a higher reactivity and more open area of mRNA. the data are stored in 2 whole transcriptome bedgraph files adding to 50 MB.</p>
<h3> m6a </h3>
  <p> m6a data detects nucleotides with the m6a modification. values are -log10 p-values, thus, bars extending further down the graph are more confident sites of m6a modification. blue and red coding are for replicate 1 and 2 respectively. files are stored as compressed bed files adding to 13 MB. </p>
<h3> in silico folding computation (complementarity) </p>
  <p> RNA duplex (https://www.tbi.univie.ac.at/RNA/RNAduplex.1.html) was run for each microRNA and each canonical transcript. the dG values indicate sites where complementarity was detected, with a more negative value indicating a stronger complementarity. the data is stored in tsv files for each gene, with columns for each miRNA. the CYP3A4 file is 5.4 MB </p>

<h3> the x-axis </h3>
  <p> all ensembl transcripts were merged so that all coordinates from any transcript isoform would be present. This is why some stretches appear blank in the plots. genetic coordinates for each gene are stored in txt files (CYP3A4 is 127 KB).  </p>
<h3> defining the UTRs </h3>
    <p> The UTRs are shown according to the canonical transcript. this is why some plots have space preceeding the 5' UTR. canonical coordinates are stored in txt files (CYP3A4 is 53 KB) </p>  


</body>
</html>
